src/resolvers/applicationCycleResolver.ts

```diff
@@ -21,10 +23,18 @@ const applicationCycleResolver:any={
                 return newapplicationCycle;
               },
               async deleteApplicationCycle(_parent:any, _args:any){
-                const deleteapplicationCycle= await applicationCycle.findById(_args.id);
-                if(!deleteapplicationCycle) throw new Error("This applicationCycle doesn't exist")
-                const deletedapplicationCycle = await applicationCycle.findByIdAndRemove(_args.id);
-                return deletedapplicationCycle;
+                      const applicationCycleToDelete= await applicationCycle.findById(_args.id);
+                      if(applicationCycleToDelete !=null){
+                          const users = await  TraineeApplicant.findOne({cycle_id:_args.id})
+                          if (users) {
+                                throw new Error("`cycle has applicant in it`") 
```

Comment:
1. `cycle has applicant in it' this string should change to
cycle has some applicants.
2. const users ===? const user (you are finding one)

Reply:
done